moveit_core/collision_detection_fcl/src/collision_common.cpp

```diff
@@ -585,8 +591,23 @@ bool distanceCallback(fcl::CollisionObjectd* o1, fcl::CollisionObjectd* o2, void
 #else
         dist_result.nearest_points[0] = Eigen::Vector3d(contact.pos.data.vs);
         dist_result.nearest_points[1] = Eigen::Vector3d(contact.pos.data.vs);
-        dist_result.normal = Eigen::Vector3d(contact.normal.data.vs);
+        Eigen::Vector3d normal(contact.normal.data.vs);
```

Comment:
#505 does not set the normal

Reply:
this looks like a bug, normal is redefined in the condition below

Reply:
also, the logic seems to be fixed in the sync